### ISSUE TITLE
jQuery datepicker doesn't have a dark mode theme

### ISSUE THREAD

**Issue by nbarry10** [unl/wdntemplates#1458]
![jquery-datepicker-dark-mode](https://user-images.githubusercontent.com/5738813/91900170-e40a0200-ec63-11ea-8510-5e2d52a48f02.png)

**Comment by skoolbus39** [comment]
The datepicker styles are updated for dark mode but there are some other jQuery UI components that need dark-mode-specific styles #1488
